rocketpy/control/controller.py

```diff
@@ -57,13 +78,16 @@ def __init__(
         -------
         None
         """
-
-        self.observed_objects = observed_objects
+        self.interactable_objects = interactable_objects
         self.controller_function = controller_function
         self.sampling_rate = sampling_rate
         self.name = name
         self.prints = _ControllerPrints
 
+        self.observed_variables = (
+            initial_observed_variables  # list of lists of observed variables
+        )
```

Comment:
May I ask, what is the purpose of the brackets here? I believe this line would have the same behavior without brackets.

Nonetheless the goal here seems to create a list of lists, therefore the line likely should be:

```py
self.observed_variables = [initial_observed_variables]
```

Then, differently from a immutable tuple, you can append new states with: `self.observed_variables.append(new_variables)`. What is happening on line 121 is that you are appending to the same list of `initial_observed_variables`.